recipe/0514-ml-usage-tags/index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: IIIF ML/AI Usage Tags Recipe
+id: 
+layout: recipe
+tags: [rights, requiredStatement]
+summary: "Ethical ML/AI Use Tags for IIIF Manifests"
+viewers:
+ - Mirador
+ - UV
+topic: 
+ - text
```

Comment:
The topics can only be one of the following

-  basic
- property
- note
- structure
- annotation
- image
- AV
- realWorldObject
- geo-recipes
- content-state

I think I would go for `note` at the moment.

Reply:
Got it, just changed to `note`, thanks Glen.